src/js/tools/appHistory.js

```diff
@@ -2,9 +2,12 @@ import createBrowserHistory from 'history/createBrowserHistory'
 
 let appHistory = createBrowserHistory();
 
-appHistory.goTo = function(url)
-{
-    appHistory.replace(url);
+appHistory.goTo = function (url) {

```

Comment:
doesn't work. (1) Freezes page when I push('/') instead of window.scrollTo, (2) i get a warning saying i shouldn't change router history